zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -355,6 +355,10 @@ private InterpreterGroup createInterpreterGroup(String id,
       Properties properties)
       throws InterpreterException {
 
+    //When called from REST API without option we receive NPE
+    if (option == null || properties == null)
+      throw new InterpreterException("Can't create group with null option or properties");
```

Comment:
Please have individual checks for each parameter and throw nullargumentexception with the argument name so we know what arg was null.